symfony/framework-bundle/3.3/config/services.yaml

```diff
@@ -5,24 +5,23 @@ parameters:
 services:
     # default configuration for services in *this* file
     _defaults:
-        # automatically injects dependencies in your services
```

Comment:
The current "before line the line" comments make the section scary to me
I propose to put these comments on the end of each lines - looks better to the eye IMHO